static/gsApp/views/seerAutomation/repoDetails.tsx

```diff
@@ -0,0 +1,155 @@
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {IconBitbucket} from 'sentry/icons/iconBitbucket';
+import {IconGithub} from 'sentry/icons/iconGithub';
+import {IconGitlab} from 'sentry/icons/iconGitlab';
+import {IconOpen} from 'sentry/icons/iconOpen';
+import {IconVsts} from 'sentry/icons/iconVsts';
+import {t, tct} from 'sentry/locale';
+import {DEFAULT_CODE_REVIEW_TRIGGERS} from 'sentry/types/integrations';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHeader';
+
+import {useBulkUpdateRepositorySettings} from 'getsentry/views/seerAutomation/onboarding/hooks/useBulkUpdateRepositorySettings';
+import useRepositoryWithSettings from 'getsentry/views/seerAutomation/onboarding/hooks/useRepositoryWithSettings';
+
+const PROVIDER_ICONS = {
+  github: IconGithub,
+  'integrations:github': IconGithub,
+  'integrations:github_enterprise': IconGithub,
+  bitbucket: IconBitbucket,
+  'integrations:bitbucket': IconBitbucket,
+  visualstudio: IconVsts,
+  'integrations:vsts': IconVsts,
+  gitlab: IconGitlab,
+  'integrations:gitlab': IconGitlab,
+};
+
+export default function SeerRepoDetails() {
+  const {repoId} = useParams<{repoId: string}>();
+  const organization = useOrganization();
+
+  const canWrite = true;
+
+  const {
+    data: repoWithSettings,
+    error,
+    isPending,
+    refetch,
+  } = useRepositoryWithSettings({repositoryId: repoId});
+
+  const {mutate: mutateRepositorySettings} = useBulkUpdateRepositorySettings();
+
+  if (isPending) {
+    return <LoadingIndicator />;
+  }
+
+  if (error) {
+    return <LoadingError onRetry={refetch} />;
+  }
+
+  const ProviderIcon =
+    PROVIDER_ICONS[repoWithSettings?.provider?.id as keyof typeof PROVIDER_ICONS] ??
+    IconOpen;
+
+  return (
+    <Fragment>
+      <SentryDocumentTitle
+        title={t('Repository Seer Settings')}
+        projectSlug={repoWithSettings?.name}
+      />
+      <SettingsPageHeader
+        title={tct('Seer Settings for [repoName] [providerLink]', {
+          repoName: <code>{repoWithSettings?.name}</code>,
+          providerLink: (
+            <ExternalLink href={repoWithSettings?.url}>
+              <ProviderIcon size="md" />
+            </ExternalLink>
+          ),
+        })}
+        subtitle={t('Choose how Seer automatically reviews your pull requests.')}
+        action={
+          <LinkButton
+            href="https://docs.sentry.io/product/ai-in-sentry/ai-code-review/"
+            external
+          >
+            {t('Read the docs')}
+          </LinkButton>
+        }
+      />
+
+      {isSupportedAutofixProvider(repoWithSettings?.provider) ? (
+        <Form
+          saveOnBlur
+          apiMethod="PUT"
+          apiEndpoint={`/organizations/${organization.slug}/`}
```

Comment:
Is this the right endpoint?

Reply:
was wrong, fixed! and removed the useMutation hook in favour of this built-in form prop.